scraper/src/strategies/default_strategy.py

```diff
@@ -213,10 +213,12 @@ def get_records_from_dom(self, current_page_url=None):
             # http://localhost:1314/index.html#pyansys-actions
             cname = os.getenv("DOCUMENTATION_CNAME")
             if cname is not None:
-                http, _, port_and_page = record["url"].split(":", 2)
-                port = os.getenv("DOCUMENTATION_PORT")
-                path = port_and_page[len(port):]
-                record['url'] = f"{http}s://{cname}{path}"
+                url_parts = record.get("url", "").split(":", 2)
+                if len(url_parts) > 2:
+                    http, _, port_and_page = url_parts
```

Comment:
```suggestion
                url = record.get("url")
                if url:
                    http, _, port_and_page = url.split(":", 2)
```

Reply:
Let's do it like this better

Reply:
We're facing an issue where the 'DOCUMENTATION_CNAME' variable in our workflow conflicts with the CNAME in the CLI command. Sometimes, when we try to split it, things go wrong. To fix this, we need to make sure the variable always has a consistent format.

Reply:
The issue is not here... The issue is whay are you using the DOCUMENTATION_CNAME env variable when you are accepting as an input the "cname"?

Reply:
You should stick to only the input CNAME and not use DOCUMENTATION_CNAME

Reply:
No we are not accepting here  cname as input. If we are accepting it is going to be a lot of refactoring expecially with this fork lib. the easiest way we found was looking at the env variable.  ( we are crawing each page with local host adding the scraped data and then renaming the local host with cname only if we are looking at the option local upload, we have other options too in which we are directly scraping from url
)

Reply:
Then, why not use a different env variable...

``SCRAPER_DOCUMENTATION_CNAME``

You need to avoid clashes

Reply:
Adapted, also adapted the port as SCRAPER_DOCUMENTATION_PORT